pkg/rancher-components/src/components/Form/LabeledInput/LabeledInput.vue

```diff
@@ -349,6 +353,7 @@ export default defineComponent({
     <slot name="field">
       <TextAreaAutoGrow
         v-if="type === 'multiline' || type === 'multiline-password'"
+        :id="hasLabel ? labelForIdentifier : ''"
```

Comment:
Each instance of `labelForIdentifier` needs to be replaced if we're going to use `inputId` for the data prop.

Reply:
fixed